website/docs/getting-started/installation.mdx

```diff
@@ -20,42 +20,97 @@ import TOCInline from '@theme/TOCInline';
 <TOCInline toc={toc} />
 ```
 
-## Option A: Compiling Locally (Recommended)
+## Option A: Download Pre-Built Releases (Recommended)
 
-### Installing Dependencies
+### a) Pre-Built Binary
+
+1. Go to the [latest release on GitHub](https://github.com/moves-rwth/caesar/releases/latest) and download the release for your operating system.
+
+2. Extract the `caesar` binary from the downloaded archive.
+
+:::info Additional Steps
```

Comment:
This box visually very striking, but irrelevant for everyone not on MacOS. Just add a third step with a sentence just for MacOS.

Reply:
I agree, that its color is forcing too much attention to it. Maybe a `:::note` (grey) is better in this context?

I disagree on just adding it as a third step to the overall procedure, with the same reasons as in our earlier discussion. It is not a necessary step for every Mac user. It is merely a hint which might help with getting the tool running. As such I would prefer to keep it separate from the 'main instructions'. The current layout is also suitable to cover potential future hurdles on other systems. 
- One could remove the tabs for Debian/Linux and Windows, as they are just placeholders. My worry was that this causes confusion on the readers site, as the selected tab is synced on the whole page. A user might wonder whether their browser is malfunctioning and they are missing out on essential information.
- Wrapping all steps in the platform-dependent tabs (with 1+2 being the same everywhere) would be an alternative but causes a lot of useless duplication.

I just tried the one-tab option again. And I think it looks better than expected (reasonable and not confusing). So I would propose using the single-tab option (as it effectively highlights that this is a macOS-specific step), together with the grey `note` admonition. Without the box, the indentation of the tab(s) seems off.